src/core/api/__tests__/public_api.test.ts

```diff
@@ -0,0 +1,571 @@
+      });
+    });
+
+    describe("setAudioBitrate/getManualAudioBitrate", () => {
+      it("should have a -1 manual audio bitrate by default", () => {
+        expect(player.getManualAudioBitrate()).to.equal(-1);
+      });
+
+      it("should update manual audio bitrate when calling setAudioBitrate", () => {
+        const oldManual = player.getManualAudioBitrate();
+
+        player.setAudioBitrate(84);
+        expect(player.getManualAudioBitrate()).to.equal(84);
+        player.setAudioBitrate(-1);
+        expect(player.getManualAudioBitrate()).to.equal(-1);
+        player.setAudioBitrate(0);
+        expect(player.getManualAudioBitrate()).to.equal(0);
+
+        player.setAudioBitrate(oldManual);
+        expect(player.getManualAudioBitrate()).to.equal(oldManual);
+      });
+    });
+
+    describe("setVideoBitrate/getManualVideoBitrate", () => {
+      it("should have a -1 manual video bitrate by default", () => {
+        expect(player.getManualVideoBitrate()).to.equal(-1);
+      });
+
+      it("should update manual video bitrate when calling setVideoBitrate", () => {
+        const oldManual = player.getManualVideoBitrate();
+
+        player.setVideoBitrate(84);
+        expect(player.getManualVideoBitrate()).to.equal(84);
+
+        player.setVideoBitrate(-1);
+        expect(player.getManualVideoBitrate()).to.equal(-1);
+
+        player.setVideoBitrate(0);
+        expect(player.getManualVideoBitrate()).to.equal(0);
+
+        player.setVideoBitrate(oldManual);
+        expect(player.getManualVideoBitrate()).to.equal(oldManual);
+      });
+    });
+
+    describe("setMaxVideoBitrate/getMaxVideoBitrate", () => {
+      /* tslint:disable:max-line-length */
+      it("should update the maximum video bitrate when calling setMaxVideoBitrate by default", () => {
+      /* tslint:enable:max-line-length */
+        const oldMax = player.getManualVideoBitrate();
+
+        expect(player.setMaxVideoBitrate(Infinity)).to.equal(undefined);
+        expect(player.getMaxVideoBitrate()).to.equal(Infinity);
+
+        expect(player.setMaxVideoBitrate(500)).to.equal(undefined);
+        expect(player.getMaxVideoBitrate()).to.equal(500);
+
+        expect(player.setMaxVideoBitrate(3)).to.equal(undefined);
+        expect(player.getMaxVideoBitrate()).to.equal(3);
+
+        expect(player.setMaxVideoBitrate(Infinity)).to.equal(undefined);
+        expect(player.getMaxVideoBitrate()).to.equal(Infinity);
+
+        expect(player.setMaxVideoBitrate(oldMax)).to.equal(undefined);
+        expect(player.getMaxVideoBitrate()).to.equal(oldMax);
+      });
+    });
+
+    describe("setMaxAudioBitrate/getMaxAudioBitrate", () => {
+      /* tslint:disable:max-line-length */
+      it("should update the maximum audio bitrate when calling setMaxAudioBitrate by default", () => {
+      /* tslint:enable:max-line-length */
+        const oldMax = player.getManualAudioBitrate();
+
+        expect(player.setMaxAudioBitrate(Infinity)).to.equal(undefined);
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Remove this use of the output from "setMaxAudioBitrate"; "setMaxAudioBitrate" doesn't return anything. ([typescript:S3699](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS3699))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=437&issues=AWgOIuUCUMkE2J581753&open=AWgOIuUCUMkE2J581753)

[ISSUE_KEY]: AWgOIuUCUMkE2J581753